src/mcp/tools/core/index.ts

```diff
@@ -5,11 +5,13 @@ import { get_project } from "./get_project.js";
 import { use_project } from "./use_project.js";
 import { get_sdk_config } from "./get_sdk_config.js";
 import { list_apps } from "./list_apps.js";
+import { init } from "./init.js";
 
 export const coreTools: ServerTool[] = [
   get_project,
   use_project,
   list_apps,
   get_sdk_config,
   consult_assistant,
+  init,
```

Comment:
Are you confident in landing additional products in `init` before release cut on Thursday? I don't want to release init and it only works for one product, would rather leave commented out for now if so